### PR TITLE
updating Composites and Components sections in Storybook Overview page to be more visual

### PR DESCRIPTION
**Pull request by alcail**
# What
added visual explanations to Composites and Components sections in Storybook Overview page

# Why
Too much text there
task https://skype.visualstudio.com/SPOOL/_workitems/edit/2641270

# How Tested
ran Storybook locally

* Composites
  image
  image

* Components
  image
  image

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/Overview.stories.mdx [hunk 1]
```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
...
+  </Stack.Item>
+  <Stack.Item align="center">
+    <img
+      src="images/callComposite_CamerasOn.png"
+      alt="Call composite where some participants have their camera turned on"
+    />
+  </Stack.Item>
+</Stack>
+
+For Teams Interop the experience includes lobby functionality for user to wait to be admitted.
```
Comment (JamesBurnside) [inline packages/storybook/stories/Overview.stories.mdx:95]: > For Teams Interop

Link here to something that details what teams interop is, e.g. `./?path=/docs/examples-teamsinterop-compliancebanner--compliance-banner`

Comment (JamesBurnside) [reply packages/storybook/stories/Overview.stories.mdx:95]: nit: consider `experience` over `functionality`, e.g. `When joining a Microsoft Teams call, the Call Composite includes a lobby experience while the user is waiting to be admitted into the call.`

File: packages/storybook/stories/Overview.stories.mdx [hunk 2]
```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
...
+devices, participate in the call with video and see other participants, including those with video turn on.
+
+Here are visual examples of Call composite where participants have their camera turned off, then some ahve them turned on.
+People speaking with microphone unmuted will have their tile appearing with a blue border.
+
+<Stack
+  horizontal
+  horizontalAlign="space-between"
+  tokens={{ childrenGap: '2rem' }}
+  style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}
```
Comment (JamesBurnside) [inline packages/storybook/stories/Overview.stories.mdx:79]: pull out this background style color code as a constant somewhere as it's used in multiple places and will want to always be consistent

**Comment by JamesBurnside** [comment]
nit: for this I think a person's name would be better to make it clear that the text is the person's name:
image

File: packages/storybook/stories/Overview.stories.mdx [hunk 3]
```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
...
+    <img
+      src="images/callComposite_TeamsInterop_AdmissionFromTeams.png"
+      alt="Popup in Teams to admit ACS participants"
+    />
+  </Stack.Item>
+</Stack>
+
+### Chat composite
+
+[ChatComposite](./?path=/story/composites-chat-basicexample--basic-example) is providing achat experience where user can send and receive
```
Comment (JamesBurnside) [inline packages/storybook/stories/Overview.stories.mdx:122]: achat -> a chat

File: packages/storybook/stories/Overview.stories.mdx [hunk 4]
```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
...
+messages. Thread events like typing, reads, participants entering and leaving are displayed to the user as part of the chat thread.
```
Comment (JamesBurnside) [inline packages/storybook/stories/Overview.stories.mdx:123]: `reads` doesn't work here, consider:
`Message thread events such as typing indicators, whether the message has been successfully delivered or read, or participants entering and leaving the chat are...`

File: packages/storybook/stories/Overview.stories.mdx [hunk 5]
```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
...
+
+Here are visual examples of Chat composite with or without participant list.
```
Comment (JamesBurnside) [inline packages/storybook/stories/Overview.stories.mdx:125]: As participant list is in beta (and won't be in the GA package) we should probably remove it from these screenshots

File: packages/storybook/stories/Overview.stories.mdx [hunk 6]
```diff
@@ -64,28 +64,128 @@ npm i --save @azure/communication-react
...
+| [Message Thread](./?path=/story/ui-components-messagethread--message-thread)                              | Container that renders chat messages, system messages, and custom messages    |
+| [Send Box](./?path=/story/ui-components-sendbox--send-box)                                                | Text input component with a discrete send button                              |
+| [Message Status Indicator](./?path=/story/ui-components-messagestatusindicator--message-status-indicator) | Multi-state message status indicator component to show status of sent message |
+| [Typing indicator](./?path=/story/ui-components-typingindicator--typing-indicator)                        | Text component to render the participants who are actively typing on a thread |
+
+### Common Components
+
+<Stack style={{ backgroundColor: '#EFF6FC', padding: '2.25rem' }}>
+  <Stack.Item align="center">
+    <img src="images/components_common.png" alt="example of components that can be used for chat" />
```
Comment (JamesBurnside) [inline packages/storybook/stories/Overview.stories.mdx:181]: As participant pane is in beta I think we should avoid references on the homepage to participant pane being in chat

**Comment by JamesBurnside** [comment]
Can we remove Overview from the headings?
image

And just have "Composites"
"UI Components"

In a seperate PR I think we should also remove this big chunk of text at the start so these new composite screenshots can be seen as soon as you land on the page:
image
(can leave one or two lines as an intro)

**Comment by JamesBurnside** [comment]
Any work to add these to the Get Started page of the Composites? 
image
It feels really lacking now

**Review by JamesBurnside — COMMENTED** [review]
These changes are great! Left some suggestions but can feel free to ignore them - looks so much better already!

**Comment by alcail** [comment]
> nit: for this I think a person's name would be better to make it clear that the text is the person's name: image
> 
> (and then remove "with name "I'm joing from ACS" from the text above)

done :)

**Comment by alcail** [comment]
> ork to add these/similar to the Get Started page of the Composites?

Not that I know but we always can create a task for it ;)

**Review by anjulgarg — APPROVED** [review]
Beautiful

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**